.travis.yml

```diff
@@ -26,6 +26,9 @@ matrix:
     - php: 5.4
       env: COVERALLS=1 DEFAULT=0 DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
 
+  allow_failures:
+    - php: 7.0
```

Comment:
It shouldn't be under allow_failures. Core runs fine on php 7 so this plugin should too.